functions/go-mongo/go.mod

```
@@ -0,0 +1,22 @@
+module github.com/scaleway/serverless-examples/functions/go-mongo
+
+go 1.24.1
```

```suggestion
go 1.24
```

Otherwise, this might break if someone tries to use another patch version, depending on the Go runtime under the hood.